packages/web/components/bridge/immersive/use-bridge-quotes.ts

```diff
@@ -160,6 +167,7 @@ export const useBridgeQuotes = ({
         },
         {
           enabled:
+            !isTxPending &&
```

Comment:
Prevents queries from running while the tx is confirming/just confirmed

Reply:
Let's add this comment here too